### DISCUSSION TITLE
windows cursor still visible in Osu!, clicking tabs out.

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#20175] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Note: I am reporting this issue for someone else. (they couldn't sign up for github.)

When the cursor should be in the application, The cursor appears double, with the OS cursor still visible alongside the Osu! cursor. upon clicking while both cursors are visible, You are immediately tabbed out of Osu!. This should be normal behavior when the cursor is outside of the window, but the cursor is inside. I believe this is an issue with conflicting mouse inputs because the Osu! cursor moves slower than the OS cursor, and this is because the sensitivity in Osu! is lower. when high precision mouse is turned OFF, or sensitivity is set to 1.00, then the Osu! and OS cursors stack on top of each other. so It's probably conflicting mouse inputs.

### Screenshots or videos

Double cursor
![image](https://user-images.githubusercontent.com/78441708/188799618-e772a832-199f-4876-9053-4a2625b5f5b4.png)

cursors moving at different speeds
![image](https://user-images.githubusercontent.com/78441708/188800797-49996e70-399f-4630-ab17-7ef9e7c5cca8.png)
![image](https://user-images.githubusercontent.com/78441708/188800870-8fbeeea0-c6de-493a-935d-53170b99baba.png)
![image](https://user-images.githubusercontent.com/78441708/188800903-3cf5e8dd-c662-44c5-9c6d-eb35560e4281.png)
![image](https://user-images.githubusercontent.com/78441708/188800917-1d5c57ce-73f3-4d6b-98ae-64919ab696b8.png)

again, clicking while both cursors are visible tabs out of Osu!.

### Version

2022.902.1-lazer

### Logs

[logs.zip](https://github.com/ppy/osu/files/9503225/logs.1.zip)

**Comment by peppy** [comment, marked as answer]
This is quite likely going to be something running outside of the game. Can you confirm what the "+" is on the photos(?) provided? What version of windows? What software is installed?

**Reply by Axilotl17** [reply]
sorry, "+" is a custom cursor image, i believe it should work the same as a normal cursor, although ill follow up about that. windows 10, and I'm not sure what you mean by "what software"

**Reply by Axilotl17** [reply]
yeah turning off the custom cursor thing fixed it .-. i feel dumb not thinking of that lol

**Reply by peppy** [reply]
If it was done using the windows cursor settings, it should not cause any issues, so I'm a bit curious why this fixed anything. How was the custom cursor being loaded? Was any external software involved?

**Reply by Axilotl17** [reply]
yes external software
